AutomatticTracks/src/main/java/com/automattic/android/tracks/crashlogging/internal/SentryErrorTrackerWrapper.kt

```diff
@@ -15,14 +14,6 @@ internal class SentryErrorTrackerWrapper {
         }
     }
 
-    fun clearBreadcrumbs() {
-        Sentry.clearBreadcrumbs()
-    }
-
-    fun setUser(user: User?) {
-        Sentry.setUser(user)
-    }
```

Comment:
Friendly reminder: the user is know set in `BeforeSendCallback` https://github.com/Automattic/Automattic-Tracks-Android/pull/72/files#diff-92307566b0dd043337c1aa764dfc72fcb338e57770bd7ffae0a6e52a250fa4f9R50